Update reference docs (#1357)
tinacms/tinacms.org: content/docs/reference/collections.md

---
title: Collections
id: collections
last_edited: '2021-07-27T15:51:56.737Z'
next: /docs/reference/fields
prev: /docs/reference/schema
---

> Collections represent a type of content (EX, blog post, page, author, etc). It is recommended to use singular naming in a collection.

Reviewer: ```suggestion
Collections represent a type of content (EX, blog post, page, author, etc). We recommend using singular naming in a collection.
```

| Property     | Description              |
|--------------|--------------------------|
| `name` | The name of the collection |

> | `path` | The path to a folder of where the content is stored. (relative to where the CLI is running) |

Reviewer: ```suggestion
| `path` | The path (relative to where the CLI is running) to a folder where the content is stored. |
```
